chapter2/tests/t6/chapter2-l2-t6-get-file-unique-dirs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Task 6 - get only dirs of the files
+set -eo pipefail
+
+
+# Function for printing error messages and exiting with a specific exit code
+# $1 - The exit code to use when exiting the script
+# $2 - The error message to print
+err() {
+  local code="$1"
+  shift
+  echo "[[ERROR]: $(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+  exit "$code"
+}
+
+
+# Args check
+if [ "$#" -ne 1 ]; then
+  err 255 'Two arguments are required.'
+fi
+
+
+# Find only regular files and put in array
+files=$(find "$1" -type f)
+
+
+# Leave only a list of all unique directories, without files
+find "$1" -type f | while read file; do
+  echo "${file%/*}"
+done | sort -u
```

Comment:
Add 1 empty line at the end of each script

Reply:
> Add 1 empty line at the end of each script

1 empty line exist, no error..

Reply:
Where? https://github.com/rtest12/devops-camp-2023/blob/266bff33a3e4226a8925fe6fd4ed1a40e8d5c73f/chapter2/tests/t6/chapter2-l2-t6-get-file-unique-dirs.sh